### PR TITLE
Gets vine video ids

### PR DESCRIPTION
**Pull request by kaganjd**
Hey, I've been working on a [project at Mozilla](github.com/meandavejustice/min-vid) that uses this library. We added vine support, so I thought I'd upstream the vine code, if you're interested. I've included tests for the vine URLs I've seen in the wild, but if you know of any I've missed, let me know. I'm happy to update the tests.

### CODE REVIEW

**Comment by radiovisual** [comment]
Awesome! Thanks! I really appreciate it!

File: index.js [hunk 1]
```diff
@@ -39,6 +41,17 @@ function vimeo(str) {
 }
 
 /**
+ * Get the vine id.
+ * @param {string} str - the url from which you want to extract the id
+ * @returns {string|undefined}
+ */
+function vine(str) {
+	const regex = /https:\/\/vine\.co\/v\/([a-zA-Z0-9]*)\/?/;
```
Comment (radiovisual) [inline index.js]: because I am targeting all (including early) node versions, I will have to use `var` instead of `const`

File: readme.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # get-video-id 
 
-> Get the Youtube or Vimeo video id from a url or embed string.
+> Get the Youtube, Vimeo, or Vine video id from a url or embed string.
 
 [![Build Status](https://travis-ci.org/radiovisual/get-video-id.svg?branch=master)](https://travis-ci.org/radiovisual/get-video-id) [![Coverage Status](https://coveralls.io/repos/github/radiovisual/get-video-id/badge.svg?branch=master)](https://coveralls.io/github/radiovisual/get-video-id?branch=master)
 
-This module will extract the Youtube or Vimeo Video ID from any known Youtube or Vimeo url pattern, including embed strings.
+This module will extract the Youtube, Vimeo, or Vine Video ID from any known Youtube or Vimeo url pattern, including embed strings.
 *If you discover a Youtube or Vimeo url pattern that is not covered by this module, please [open an issue](https://github.com/radiovisual/get-video-id/issues)
```
Comment (radiovisual) [inline readme.md]: We can update this line to include Vine urls as well.

File: index.js (hunk 1 above)
Comment (kaganjd) [inline index.js]: Got it! Made this change.

File: readme.md (hunk 2 above)
Comment (kaganjd) [inline readme.md]: Good catch. Done.

**Comment by radiovisual** [comment]
Awesome! Thanks a lot for your contribution! I really appreciate it!

**Comment by radiovisual** [comment]
merged! 

bill-hader-do-want-11Sp47cNKpYQ00

**Comment by radiovisual** [comment]
FYI: @kaganjd , I have been meaning to update this module to a 1.0 release, since I am pretty sure the API is where I want it to be, so I rolled this PR into the version 1.0 release. I just published it to npm, so your PR has been included in version `1.0` 🎈

**Comment by kaganjd** [comment]
That's great @radiovisual. Congrats, and thanks for the update and the module. Really helpful for our project.

**Pull request merged**